### DISCUSSION TITLE
Our first voting period starts!

### DISCUSSION THREAD

**Discussion by ruphy** [publiccodeyml/publiccode.yml#150] [Announcements]
Dear all,

as [detailed in our procedures](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/procedure-proposing-changes-and-voting.md), it is my duty and pleasure to start the voting phase for a new release of publiccode.yml.

I am particularly excited as this will be the first one we release together. For this reason I would like to exceptionally extend the voting period to two weeks.

The rules are simple:
- Voting is only open to all members of the Steering Committee
- Voting happens on formal proposals opened as Pull Requests, once the vote is open on a proposal, a bot will create a message. A thumbs up to that message is considered an approve vote, while a thumbs down is considered
- Only approve or reject votes are used to determine the results.
- In general, proposals require unanimity to pass, but when that doesn’t happen, proposals with at least 50% approve votes get discussed for an additional period of minimum of 90 days, after which they are put again to vote. In this second round, the change only needs 75% approve votes in order to be considered accepted. [More details](https://github.com/publiccodeyml/publiccode.yml/blob/main/governance/procedure-proposing-changes-and-voting.md).

We'll vote on the following Proposals:
https://github.com/publiccodeyml/publiccode.yml/pull/88
https://github.com/publiccodeyml/publiccode.yml/pull/97
https://github.com/publiccodeyml/publiccode.yml/pull/139
https://github.com/publiccodeyml/publiccode.yml/pull/141
https://github.com/publiccodeyml/publiccode.yml/pull/144
https://github.com/publiccodeyml/publiccode.yml/pull/145
https://github.com/publiccodeyml/publiccode.yml/pull/146
https://github.com/publiccodeyml/publiccode.yml/pull/147
https://github.com/publiccodeyml/publiccode.yml/pull/148
https://github.com/publiccodeyml/publiccode.yml/pull/149

All votes will need to be cast by the 28th of April at 00.01 UTC.

I’m looking forward to tagging this release!
Riccardo

cc @zorp @bzg @cknebel @ShimonShore

**Comment by ruphy** [comment]
for the steering commitee, you can also see the list of PRs on which you should vote here: https://github.com/publiccodeyml/publiccode.yml/labels/vote-start

**Comment by ruphy** [comment]
@zorp @bzg @cknebel @ShimonShore is everything going well with the voting? you just need to add a 👍  or 👎  on the bot comment for the various pull requests. the period ends in ±48 hours, let me know if I can assist in any way

**Comment by bzg** [comment]
Thanks for the heads up, I noted the deadline and will try to vote  in time.

**Reply by bzg** [reply]
Done, thanks!
